Unit.h

```diff
@@ -65,4 +65,5 @@ class Unit
 
     /// attack the other unit and fight until one of them is dead
     virtual void Attack(Unit &targetUnit /** [in]*/);
+	static void Fight(Unit& first, Unit& second);
```

Comment:
OK... Mondjuk igazán lehetett volna ez is egy egyparaméteres non-static metódus, ami csak a célpontot várja, meg dokumentálva sincs, de most egynek elmegy.